tests/etl/test_perf_data_adapters.py

```diff
@@ -41,6 +41,10 @@ def test_adapt_and_load():
         # one extra result for the summary series
         result_count += (len(datum['blob']["results"]) + 1)
 
+        # we create one per counter
+        if 'talos_counters' in datum['blob']:
+            result_count += (len(datum['blob']["talos_counters"]))
```

Comment:
Oops, I guess it was like this before. Might as well change it while you're there though. :)